backend/src/middleware/auth.js

```
@@ -0,0 +1,344 @@
+      if (!resourceId) {
+        return res.status(400).json({ 
+          error: 'Resource ID required',
+          message: `${resourceType} ID is required`
+        });
+      }
+
+      // Import services dynamically to avoid circular dependencies
+      let service;
+      switch (resourceType) {
+        case 'fish':
+          const { FishService } = await import('../services/fishService.js');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)